composer.json

```diff
@@ -38,6 +38,7 @@
         "illuminate/routing": "^9 || ^10",
         "illuminate/support": "^9 || ^10",
         "illuminate/validation": "^9 || ^10",
+        "google/protobuf": "^3.21",
```

Comment:
Let's add that to the dev-dependencies and to `suggest` as well, adding new deps is a breaking change.